doc/source/whatsnew/v0.19.0.txt

```diff
@@ -377,6 +377,20 @@ Other enhancements
 
     pd.Timestamp(year=2012, month=1, day=1, hour=8, minute=30)
 
+- the ``.resample()`` function now accepts a ``on=`` or ``level=`` parameter for resampling on a column or ``MultiIndex`` level (:issue:`13500`)
```

Comment:
again would say datetimelike as well. Further I think the doc-string of ``.resample`` needs more specific wording (mentiioning datetimelike).

what we have now
```
Convenience method for frequency conversion and resampling of regular
time-series data.
```